pom.xml

```diff
@@ -1416,58 +1304,84 @@
                     </executions>
                 </plugin>
                 <plugin>
-                    <groupId>net.revelc.code</groupId>
-                    <artifactId>impsort-maven-plugin</artifactId>
-                    <version>${version.impsort-maven-plugin}</version>
+                    <groupId>com.diffplug.spotless</groupId>
+                    <artifactId>spotless-maven-plugin</artifactId>
```

Comment:
We probably need two execution mode: one for dev (auto formats) and one for CI (checks only, no auto format)